build-tools/packages/build-tools/src/bumpVersion/gitRepo.ts

```diff
@@ -184,6 +184,14 @@ export class GitRepo {
         return results.split("\n").filter(t => t !== undefined && t !== "" && t !== null);
     }
 
+    /**
+     * @returns An array containing all the modified files in the repo.
```

Comment:
Nit: Personally, I think it's potentially better to use the raw description field for comments that only specify what gets returned, rather than using an `@returns` block. That way intellisense / generated API docs, etc. see something useful as a summary. E.g. something as simple as:
```suggestion
     * Returns an array containing all the modified files in the repo.
```

Reply:
Good idea.